integration-tests/src/test/java/tachyon/master/lineage/LineageMasterIntegrationTest.java

```diff
@@ -142,7 +142,7 @@ public void lineageCompleteAndAsyncPersistTest() throws Exception {
           fileInfo.getPersistenceState());
 
       // sleep and wait for worker to persist the file
-      CommonUtils.sleepMs(5);
+      CommonUtils.sleepMs(200);
```

Comment:
for the sake of non-flaky tests, it would be nice if the thread creation was control by a heartbeat executor ... that is probably outside of the scope of this PR though